contracts/OpenStream.sol

```diff
@@ -20,6 +29,35 @@ contract OpenStream is ReentrancyGuard, IOpenStream {
         payee = _payee;
         token = _token;
         rate = _rate;
+        createdAt = block.timestamp;
+        lastClaimedAt = createdAt;
+    }
+    
+    ///@dev it gets token balance of the smart contract.
+    function getTokenBanance() public view returns (uint256) {
+        return IERC20(token).balanceOf(address(this));
+    }
+
+    ///@dev it calculates redeemed amount.
+    function calculate(uint256 claimedAt) public view returns (uint256) {
+        uint256 elapsed = claimedAt - lastClaimedAt;
+        return elapsed * rate / 30 / 24 / 3600;
+    }
+
+    ///@dev payee can claim tokens which is proportional to elapsed time (exactly seconds).`
+    function claim() external {
```

Comment:
@accompany1205 The contract will receive USDT/USDC, not native tokens

Reply:
@georgeciubotaru I was confused

Reply:
@georgeciubotaru I was confused